omero/sysadmins/system-requirements.txt

```diff
@@ -8,27 +8,41 @@ OMERO.server
 ^^^^^^^^^^^^
 
 The system requirements for OMERO.server vary greatly depending on image
-size and number of users. At a minimum we suggest:
+size and number of users. The minimum requirements should be easily exceeded
+by any recently bought hardware.
 
--  Single core 1.33GHz Intel or AMD CPU
--  2GB RAM
--  500MB of hard drive space for OMERO.server distribution
-
-The recommended OMERO.server specification we suggest for between 25-50
-users is:
+An OMERO.server specification for between 25-50 users might be:
 
 -  Quad core 1.33GHz Intel or AMD CPU
 -  8GB RAM
 -  500MB hard drive space for OMERO.server distribution
 -  Hard drive space proportional to the image sizes expected (likely
    between 10 and 100TB)
 
+A specification for a server future-proofed for 3-4 years might be:
+
+-  dual Intel Xeon Processor E5-2637 v3 4C 3.5GHz 15MB 2133MHz 135W
+-  256GB RAM
+-  2 x 200GB SSD RAID1 for OS
+-  2 x 400GB SSD RAID1 for PostgreSQL DB
```

Comment:
In which case perhaps you could state that this is the server spec excluding the repository and reference the Storage section below?